pkg/copier/copier_test.go

```diff
@@ -353,8 +353,7 @@ func TestCopierFromCheckpoint(t *testing.T) {
 	assert.NoError(t, err)
 
 	// Open chunker at the specified watermark
-	highPtr := table.NewDatum(t1new.MaxValue().Val, t1.MaxValue().Tp)
-	err = chunker.OpenAtWatermark(lowWatermark, highPtr, 0)
+	err = chunker.OpenAtWatermark(lowWatermark)
```

Comment:
This has to be pushed down into the chunker, since the high pointer needs to be calculated per table.